web-client/src/presenter/actions/getSentWorkItemsForSectionAction.js

```diff
@@ -0,0 +1,10 @@
+export default async ({ applicationContext }) => {
+  const user = applicationContext.getCurrentUser();
+  let workItems = await applicationContext
```

Comment:
Should this have a test?

Reply:
this action is actually covered, either in a unit tests which coverages a sequence, or one of our e2e tests: 
<img width="859" alt="screen shot 2019-02-18 at 5 30 06 pm" src="https://user-images.githubusercontent.com/1868782/52979293-eae36380-33a2-11e9-8547-21ba558347c5.png">

Reply:
Cool, thanks for pointing that out.